spec/decorators/sets_attributes_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe SetsAttributes do
+  subject { SetsAttributes.new(term) }
+  let(:term) { term_mock }
+  let(:test_param) {{
+    "label" => ["blah"]
+  }}
+  before do
+    stub_repository
+    allow(term).to receive(:attributes=)
+    allow(term).to receive(:attributes).and_return(test_param)
+    allow(term).to receive(:valid?).and_return(true)
+  end
+
+  describe "#set_languages" do
```

Comment:
Yeah but I think I have to do it in a different test. Probably a controller spec. This is using mocks and doubles so Id have to set up a lot more state to get the graph and stuff. Since we are using a mock for term.